tinkergraph-gremlin/src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerFactory.java

```diff
@@ -149,6 +149,13 @@ public static void generateKitchenSink(final TinkerGraph graph) {
         g.addV("loops").property(T.id, 1000).property("name", "loop").as("me").
           addE("self").to("me").
           iterate();
+        final String LABEL = "message_passing_test";
+        final String EDGE_LABEL = "msg_pass_test_edge";
+        final String PROPERTY_IN = "propertyin";
+        Vertex a = graph.addVertex(T.id, 2000, T.label, LABEL, PROPERTY_IN, "a");
```

Comment:
all of these sample graphs need a "name" property of some sort. that's why the build is currently failing for me.  please "final" the two `Vertex` variables.